Cargo.toml

```
@@ -15,11 +15,13 @@ default-run = "scd4x-util"
 [features]
 scd41 = []
 util = [ "structopt", "simplelog", "humantime", "linux-embedded-hal", "anyhow" ]
+embedded-hal-async = [ "dep:embedded-hal-async", "sensirion-i2c/embedded-hal-async"]
 default = []
 
 [dependencies]
-sensirion-i2c = "0.3.0"
+sensirion-i2c = "0.4.0"
 embedded-hal = "1.0.0"
+embedded-hal-async = { optional = true }
 
 log = { version = "0.4.16", default_features = false }
 defmt = { version = "0.3.0", optional = true }
```

```suggestion
defmt = { version = "0.3.0", optional = true }
thiserror = { version = "1.0.65", optional = true }
```
I think this is what caused the errors, not sure how this passed CI on main.